001-download-vm/Makefile

```diff
@@ -0,0 +1,6 @@
+
+vm_image=QmW9on9UeKCVDb5dsTEZJQwQoH53gXUtVfWvSBhf2Thmyq/vms/opscode_ubuntu-14.04_chef-provisionerless.box
+
+download:
+	$(IPFS_BIN_PATH)/ipfs get $(vm_image)
```

Comment:
want the output in the console videos to be super simple. let's use `ipfs` and maybe add a note that users should invoke `make` with a `$PATH` that includes `ipfs` ?